### PR TITLE
(FIX) patching of parent model managers

### PR DESCRIPTION
**Pull request by dfirst**

### CODE REVIEW

**Comment by jieter** [comment]
@dfirst thanks for the pull. 

I see you amended the tests, but it's not entirely clear to me what exact case this failed. It would be nice to get a separate test case instead of amending the current tests/models. Can you update the pull to do that?

**Comment by dfirst** [comment]
> @dfirst thanks for the pull.
> 
> I see you amended the tests, but it's not entirely clear to me what exact case this failed. It would be nice to get a separate test case instead of amending the current tests/models. Can you update the pull to do that?

Fixed.

File: tests/test_translating.py [hunk 1]
```diff
@@ -179,6 +180,9 @@ class Meta:
         self.assertEqual(TestModel1.objects.custom_method(), "foo")
         self.assertIsInstance(TestModel1.objects.all(), MultilingualQuerySet)
 
+    def test_translate_abstract_model_with_custom_manager(self):
+        self.assertTrue(isinstance(app_models.Article.objects, MultilingualManager))
```
Comment (jieter) [inline tests/test_translating.py]:
```suggestion
        self.assertIsInstance(app_models.Article.objects, MultilingualManager)
```

File: tests/app/models.py [hunk 2]
```diff
@@ -17,6 +17,11 @@ class CategoryQueryset(models.QuerySet):
 
 
 class Category(models.Model):
+    """
+    Abstract Category with custom manger required
+    to test patching of parent model managers.
+    """
+
```
Comment (jieter) [inline tests/app/models.py:20]: Can you move this comment to AbstractArticle (and amend it to reflect it's new location)?

**Review by jieter — COMMENTED** [review]
That's better, thanks! I added two additional comments.

**Comment by dfirst** [comment]
Fixed

**Comment by jieter** [comment]
@dfirst great, thanks. Merging.

**Pull request merged**